lib/libvfio-user.c

```diff
@@ -1657,7 +1668,7 @@ vfu_setup_region(vfu_ctx_t *vfu_ctx, int region_idx, size_t size,
     reg->cb = cb;
     reg->fd = fd;
     reg->offset = offset;
-    LIST_INIT(&reg->sub_reg_l);
+    LIST_INIT(&reg->subregions);
```

Comment:
I still think this belongs in vfu_create_ctx

Reply:
moved